### PR TITLE
Added pneumatics to the intake

### LINKED ISSUE #27: Intake needs to be raised/lowered with pneumatics
The intake system can be lowered and raised with pneumatics. We need to add that logic to the code.
Add a DoubleSolenoid

### PR DESCRIPTION
**Pull request by acezyx**
fixes #27

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -120,6 +120,8 @@ private void configureButtonBindings() {
     m_operator.getButtonB().whileHeld(new RunCommand(() -> m_loader.ManualLoad(0.5, 0.5), m_loader));
     m_operator.getButtonX().whileHeld(new RunCommand(() -> m_intake.ManualIntake(0.5), m_intake));
     m_operator.getButtonY().whileHeld(new RunCommand(() -> m_intake.ManualIntake(-0.5), m_intake));
+    m_operator.getButtonSelect().whileHeld(new RunCommand(() -> m_intake.MoveIntakeDown(), m_intake));
+    m_operator.getButtonStart().whileHeld(new RunCommand(() -> m_intake.MoveIntakeUp(), m_intake));
```
Comment (jnallard) [inline src/main/java/frc/robot/RobotContainer.java]: For testing, we should comment this out until we're ready to actually test the pneumatic acuators.
Can we comment this out for now?

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
   /** Creates a new Climber. */
   public Climber() {
     m_extensionController = new TalonFX(Constants.ClimberExtension);
-    m_solenoid = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, Constants.SolenoidForward, Constants.SolenoidReverse);
+    m_solenoid = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, Constants.ClimberSolenoidForward, Constants.ClimberSolenoidReverse);
```
Comment (calebur) [inline src/main/java/frc/robot/subsystems/Climber.java]: We found out tonight that we're using the REV pnumiatics hub, not CTRE as we originally thought.

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 3]
```diff
@@ -8,15 +8,21 @@
...
 import frc.robot.Constants;
 
 public class Intake extends SubsystemBase {
   private TalonFX m_IntakeController;
+  private DoubleSolenoid m_solenoid;
   /** Creates a new Intake. */
   public Intake() {
     m_IntakeController = new TalonFX(Constants.Intake);
     m_IntakeController.setNeutralMode(NeutralMode.Coast);
+    m_solenoid = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, Constants.IntakeSolenoidForward, Constants.IntakeSolenoidReverse);
```
Comment (calebur) [inline src/main/java/frc/robot/subsystems/Intake.java]: Here as well (see previous comment about REV not CTRE)

**Review by calebur — CHANGES_REQUESTED** [review]

**Review by jnallard — APPROVED** [review]
Reviewed together

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (acezyx) [reply src/main/java/frc/robot/RobotContainer.java]: We did this

File: src/main/java/frc/robot/subsystems/Climber.java (hunk 2 above)
Comment (acezyx) [reply src/main/java/frc/robot/subsystems/Climber.java]: We did this

File: src/main/java/frc/robot/subsystems/Intake.java (hunk 3 above)
Comment (acezyx) [reply src/main/java/frc/robot/subsystems/Intake.java]: We did this

**Pull request merged**